marlowe-playground-client/src/NewProject/State.purs

```diff
@@ -30,16 +30,45 @@ render ::
   State ->
   ComponentHTML Action ChildSlots m
 render state =
-  div [ classes [ ClassName "new-project-container" ] ]
-    [ input [ value (state ^. _projectName), onValueChange (Just <<< ChangeProjectName) ]
-    , hr_
-    , h2_ [ text "Choose your initial coding environment" ]
-    , div [ classes [ flex, ClassName "group", ClassName "open-buttons" ] ] (map link [ Haskell, Javascript, Marlowe, Blockly ])
-    , renderError (state ^. _error)
+  div_
+    [ div [ classes [ ClassName "modal-header" ] ]
+        [ {-
+            TODO: confirm that we dont want to change the project name and remove
+            input [ value (state ^. _projectName), onValueChange (Just <<< ChangeProjectName) ]
+             TODO: create an HTML helper so all dialogs have the same header/title?
+          -} h2 [ classes [ ClassName "title" ] ] [ text "New Project" ]
```

Comment:
I think that's fair, since we have the modal

Reply:
done, removed